.github/CODEOWNERS

```
@@ -0,0 +1,27 @@
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence, these users
+# will be requested for review when someone opens a pull request.
+
+# Actions common lib folder
+actions-shared/ @segmentio/platform-components
```

```suggestion
actions-shared/ @segmentio/platform-components
```

should be build experience team. platform-components team is not active.

https://github.com/orgs/segmentio/teams/build-experience-team